Listing.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Controllers;
+use App\Models\Property2;
+use App\Models\PropertyListingModel;
+
+class Properties extends BaseController
+{
+    public function index()
+    {
+        $db = db_connect();
+        $property = new PropertyListingModel($db);
+        $props = $property->getPropertyListing(2);

```

Comment:
Since we'll be working with APIs the last thing you should work on is to make this more dynamic rather than passing in a static value like '2' in this case.

Refer to the properties controller to see how this is done (they are very similar as the owner ID will be passed as the parameter)
When you're done, add a Route in the Routes.php file that will make it easier to access the urls. e.g listings/1 for listings belonging to userID 1. Refer to the snippet below

![image](https://user-images.githubusercontent.com/59659920/140173002-9cae99b8-d346-4a83-8831-9bbd399a6d2e.png)